kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/ManageSyncSchedule.vue

```diff
@@ -0,0 +1,230 @@
+  >
+    <KPageContainer>
+      <KGrid gutter="48">
+        <KGridItem>
+          <h1>{{ $tr('manageSyncTitle') }}</h1>
+        </KGridItem>
+
+        <KGridItem>
+          <p>{{ $tr('introduction') }}</p>
+        </KGridItem>
+
+        <KGridItem :layout8="{ span: 4 }" :layout12="{ span: 6 }">
+          <p>{{ $tr('homeFacility') }}</p>
+        </KGridItem>
+        <KGridItem
+          :layout="{ alignment: 'right' }"
+          :layout8="{ span: 4 }"
+          :layout12="{ span: 6 }"
+        >
+          <KButton
+            @click="deviceModal = true"
+          >
+            {{ $tr('addDevice') }}
+          </KButton>
+        </KGridItem>
+      </KGrid>
+
+      <CoreTable>
+        <template #tbody>
+          <tbody class="table">
+            <tr>
+              <th>{{ $tr('deviceName') }}</th>
+              <th>{{ $tr('Schedule') }}</th>
+              <th>{{ $tr('Status') }}</th>
+              <th></th>
+            </tr>
+            <tr>
+              <td>
+                <span>Kolibri Data Portal<br>
+                  http://dataportal.link
+                </span>
+              </td>
+              <td>Never</td>
+              <td>
+                <KIcon
+                  icon="disconnected"
+                />
+                Not connected
+              </td>
+              <td>
+                <KButton
+                  @click="editButton"
+                >
+                  edit
+                </KButton>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <span>MacOS<br>
+                  localhost:8000</span>
+
+              </td>
+              <td>Every Monday at 14:30</td>
+              <td>
+                <KIcon
+                  icon="onDevice"
+                />
+                Connected
+              </td>
+              <td>
+                <KButton
+                  @click="editButton"
+                >
+                  edit
+                </KButton>
+              </td>
+            </tr>
+          </tbody>
+        </template>
+
+      </CoreTable>
+      <KModal
+        v-if="deviceModal"
+        :title="$tr('selectDevices')"
+        size="medium"
+        submitText="CONTINUE"
+        cancelText="CANCEL"
+        @cancel="closeModal"
+        @submit="submitModal"
+      >
+        <KGrid>
+          <KGridItem
+            :layout8="{ span: 4 }"
+            :layout12="{ span: 6 }"
+          >
+            <KButton
+              appearance="basic-link"
+              @click="newAddress"
+            >
+              Add new Address
+            </KButton>
+          </KGridItem>
+        </KGrid>
+
+        <KGrid gutter="48">
+          <KGridItem
+            :layout8="{ span: 4 }"
+            :layout12="{ span: 6 }"
+          >
+
+            <KRadioButton
+              v-model="deviceModal"
+              label="LINUX"
+              value="linux"
+            >
+              <span>localhost:8040</span>
```

Comment:
The KButton label and value, and the <span> will need to be update in the future with user-specific data values, but it is okay for now to have placeholder text !